appveyor.yml

```diff
@@ -18,7 +18,7 @@ install:
   - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
   - ruby --version
   - gem --version
-  - gem install bundler
+  - gem install bundler -v 1.11.2
```

Comment:
What's the reason appveyor needs this?  This file is generated from https://github.com/rspec/rspec-dev/blob/master/travis/appveyor.yml and if this is a permanent change needed for all appveyor builds we should probably push this change to there.